core/templates/pages/contributor-dashboard-admin-page/contributor-dashboard-admin-page.import.ts

```diff
@@ -19,13 +19,12 @@
 import 'core-js/es7/reflect';
 import 'zone.js';
 
-import 'angular-ui-sortable';
 import uiValidate from 'angular-ui-validate';
 
 angular.module('oppia', [
   require('angular-cookies'), 'ngAnimate',
   'ngMaterial', 'ngSanitize', 'ngTouch', 'pascalprecht.translate',
-  'ui.bootstrap', 'ui.sortable', uiValidate
```

Comment:
Could you please explain why `ui.sortable` is no more needed?

Reply:
i have replaced this with cdkDropList `Angular material`

Reply:
https://material.angular.io/cdk/drag-drop/overview